lambdas/user-list-search-corpus-indexing/src/index.ts

```diff
@@ -34,13 +38,16 @@ import { removeApprovedItem } from './commands/RemoveItem';
  */
 export async function processor(event: SQSEvent): Promise<SQSBatchResponse> {
   const validPayloads: Array<EventPayload> = event.Records.map((record) => {
-    const message = JSON.parse(JSON.parse(record.body).Message);
+    const event = sqsEventBridgeEvent(record);
+    if (event == null || !validDetailTypes.includes(event['detail-type'])) {
```

Comment:
if this can be null, and it really shouldn't be... then we should update the `processor` function signature